Marlin/Marlin_main.cpp

```diff
@@ -1804,15 +1805,15 @@ static void clean_up_after_endstop_or_probe_move() {
 
     if (endstops.z_probe_enabled) return;
 
+    // Make room for probe
+    do_probe_raise(Z_RAISE_BEFORE_PROBING);
```

Comment:
The sled probe might not like this. The `dock_sled(false)` handler does a move which combines X and Z together, for some reason. If this raise seems like it's fine for the docking sled too, then I guess we can get rid of all the Z raises in `dock_sled()`. That should be just fine, because `dock_sled` is no longer called directly anywhere. Everything calls `deploy`/`stow_z_probe` now.